src/pages/saksbehandling/søknadsbehandling/fast-opphold-i-norge/FastOppholdINorge.tsx

```diff
@@ -81,114 +64,119 @@ const FastOppholdINorge = (props: VilkårsvurderingBaseProps) => {
+                                    />
+                                </RadioGruppe>
+                            )}
+                        />
                         <div className={sharedStyles.textareaContainer}>
-                            <Textarea
-                                label={intl.formatMessage({ id: 'input.label.begrunnelse' })}
+                            <Controller
+                                control={form.control}
                                 name="begrunnelse"
-                                feil={formik.errors.begrunnelse}
-                                value={formik.values.begrunnelse ?? ''}
-                                onChange={(e) => {
-                                    formik.setValues({
-                                        ...formik.values,
-                                        begrunnelse: e.target.value ? e.target.value : null,
-                                    });
-                                }}
+                                render={({ field, fieldState }) => (
+                                    <Textarea
+                                        label={formatMessage('input.label.begrunnelse')}
+                                        {...field}
+                                        feil={fieldState.error?.message}
+                                        value={field.value ?? ''}
+                                    />
+                                )}
                             />
                         </div>
                         {pipe(
                             lagreBehandlingsinformasjonStatus,
                             RemoteData.fold(
                                 () => null,
-                                () => (
-                                    <NavFrontendSpinner>
-                                        {intl.formatMessage({ id: 'display.lagre.lagrer' })}
-                                    </NavFrontendSpinner>
-                                ),
+                                () => <NavFrontendSpinner>{formatMessage('display.lagre.lagrer')}</NavFrontendSpinner>,
                                 () => (
                                     <AlertStripe type="feil">
-                                        {intl.formatMessage({ id: 'display.lagre.lagringFeilet' })}
+                                        {formatMessage('display.lagre.lagringFeilet')}
                                     </AlertStripe>
                                 ),
                                 () => null
                             )
                         )}
                         <Feiloppsummering
-                            tittel={intl.formatMessage({ id: 'feiloppsummering.title' })}
-                            feil={formikErrorsTilFeiloppsummering(formik.errors)}
-                            hidden={!formikErrorsHarFeil(formik.errors)}
+                            tittel={formatMessage('feiloppsummering.title')}
```

Comment:
Ser vi bruker en blanding av norsk/engelsk i disse keysa. Er det tekniske ord vs domeneord? Så her er det engelsk siden title/tittel aldri brukes som domeneord? Mens knappene ofte viser teksten lagre, slik at det blir et domeneord?

Reply:
Hehe, gode spørsmål. Her har vi nok rett og slett bare ikke vært så veldig konsekvente; tipper vi finner ganske mye blanding rundt omkring i appen 😅

Reply:
Men hva ønsker vi? (så vi i hvert fall kan være konsekvente framover)

Reply:
Sånn finger-i-været så tror jeg vil har flest ressurs-ID-er på norsk (med hederlige unntak at vi bruker f.eks. `.label` for å tydeliggjøre at det er en form-label), og syns det leser ganske ålreit. Hva tenker du?

Reply:
Enig, trenger ikke oversette tekniske termer som ikke føles naturlig å oversette